quantum/process_keycode/process_sequencer.c

```diff
@@ -0,0 +1,76 @@
+#include "process_sequencer.h"
+
+#if defined(SEQUENCER_ENABLE)
```

Comment:
These probably aren't needed, as the files should only be added to `SRC` if the makefile rule is set anyway.